src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -378,6 +381,22 @@ public static partial class RoslynInsertionTool
                     }
                 }
 
+                // ********************* Set PR to Auto-Complete *****************************
+                if (Options.SetAutoComplete)
+                {
+                    cancellationToken.ThrowIfCancellationRequested();
+                    Console.WriteLine($"Set PR to Auto-Complete");
+                    try
+                    {
+                        await SetAutoCompleteAsync(pullRequest);
```

Comment:
should this method take a CancellationToken?

Reply:
I'm mimicking the behavior of QueueValidationBuild, so yes?

Reply:
Just to be clear I'm referring to `SetAutoCompleteAsync`. It doesn't meaningfully affect things to not be able to cancel here but feels like we should be consistent. If it works as is then no big deal.

Reply:
oh, I guess I could easily add it there. To keep conistency though, should I add it to the QueueBuildPolicy too?